rpm/containerd.spec

```diff
@@ -151,9 +149,7 @@ done
 %doc README.md
 %{_bindir}/*
 %{_unitdir}/containerd.service
-%{_sysconfdir}/containerd
 %{_mandir}/man*/*
-%config(noreplace) %{_sysconfdir}/containerd/config.toml
```

Comment:
This is the only part I was looking at if this could have side-effects (file and directory no longer marked as "config"). I _think_ it should have no consequences (package no longer knows of this path, so won't interfere with it), but if someone knows if this is a concern, let me know.

Reply:
@thaJeztah removing this does have some impact on rpm users who are currently overwriting the rpm supplied config.toml with their own.  This being removed above, causes the user supplied override config.toml file to be deleted, when they upgrade to a rpm version with this change.  rpm moves the file `.rpmsave` and `.rpmnew` after installing the new version with above change.

Can we just continue shipping an empty config.toml with just `version=2` without the disabled cri plugin config ?

Reply:
@ravilr hmmm... yeah, we could partially revert that change if there's no other way; do you know if rpms allow for the change to be "ignored" (as in: don't touch the existing file?)

Reply:
according to https://rpm-packaging-guide.github.io/#files,  %config(noreplace) exists to protect local modifications to a config file, so that local edits to config files won't get lost during a subsequent upgrade.   
But, since the file is being removed from rpm, that doesn't apply here. 

IMO,  if the intention is to get rid of disabling of cri plugin, then rpm packaging here should continue shipping an empty config.toml with %config(noreplace).